ArcGISRuntimeSDKQt_CppSamples/Layers/QueryOGCAPICQLFilters/QueryOGCAPICQLFilters.qml

```diff
@@ -0,0 +1,145 @@
+// [WriteFile Name=Query_OGC, Category=Layers]
+// [Legal]
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import Esri.Samples 1.0
+import QtQuick.Controls 1.4
```

Comment:
We should only be using QtQuick.Controls 2 in our samples. They have a different look and feel and are the more modern approach. In this case, both 1 and 2 are imported, but 2 is imported last, so any controls (like Button, ComboBox) will use version 1 instead of version 2